doc/api/process.markdown

```diff
@@ -59,8 +59,9 @@ finished running the process will exit. Therefore you **must** only perform
 checks on the module's state (like for unit tests). The callback takes one
 argument, the code the process is exiting with.
 
-This event may not be fired if the process terminates due to signals like
-`SIGINT`, `SIGTERM`, `SIGKILL`, and `SIGHUP`.
+This event is only emitted when node exits explicitly by process.exit() or
+implicitly by the event loop draining. It will not, for example, be fired
```

Comment:
I'm not a big fan of "for example". I would rather see all possible situations precisely documented. Don't make the user guess whether or not what is written here applies to them or not.